pkg/api/k8s_versions.go

```diff
@@ -468,6 +468,7 @@ func getK8sVersionComponents(version string, overrides map[string]string) map[st
 	case "1.11":
 		ret = map[string]string{
 			"hyperkube":                        "hyperkube-amd64:v" + version,
+			"hyperkube-azs":                    "hyperkube-amd64:v" + version + "-azs",
```

Comment:
Please also try the scale and upgrade scenario for azure stack. 1.11.8 to 1.11.9 and 1.11.9 to 1.12.7

Reply:
these two operations worked on my dev env.